packages/shared/ReactFeatureFlags.js

```diff
@@ -180,8 +181,8 @@ export const enableInfiniteRenderLoopDetection = true;
 
 // Passes `ref` as a normal prop instead of stripping it from the props object
 // during element creation.
-export const enableRefAsProp = __NEXT_MAJOR__;
-export const disableStringRefs = __NEXT_MAJOR__;
+export const enableRefAsProp = true;
```

Comment:
I'll wait to merge this until I fix that class components ref bug. Or we can flip this to false temporarily and land the rest if I take too long.

Reply:
Or we can keep it as __EXPERIMENTAL__?